### PR TITLE
start_point renaming.

### PR DESCRIPTION
**Pull request by Jedzkie**
Renamed 'start_point' to 'start_point_renewal' and 'start_point_pre' to 'start_point_pre-re'.

### CODE REVIEW

File: conf/char-server.conf [hunk 1]
```diff
@@ -95,8 +95,8 @@ autosave_time: 60
 save_log: yes
 
 // Start point, Map name followed by coordinates (x,y)
-start_point: iz_int,97,90
-start_point_pre: new_1-1,53,111
+start_point_renewal: iz_int,97,90
+start_point_pre-re: new_1-1,53,111
```
Comment (MishimaHaruna) [inline conf/char-server.conf]: Can we stick to letters, numbers and underscores, for config entries? That way it'll cause less problems when we switch to libconfig (ongoing branch)

Comment (Jedzkie) [inline conf/char-server.conf]: Copy sir, going to change this ASAP. Im outside right now.

File: src/char/char.c [hunk 2]
```diff
@@ -5644,14 +5644,14 @@ int char_config_read(const char* cfgName)
 			save_log = config_switch(w2);
 		}
 		#ifdef RENEWAL
-			else if (strcmpi(w1, "start_point") == 0) {
+			else if (strcmpi(w1, "start_point_re") == 0) {
 				char map[MAP_NAME_LENGTH_EXT];
 				int x, y;
-				if (sscanf(w2, "%15[^,],%d,%d", map, &x, &y) < 3)
+				if (sscanf(w2, "%15[^,], %d, %d", map, &x, &y) < 3)
```
Comment (4144) [inline src/char/char.c]: Why adding spaces here and in second sscanf?

Comment (Jedzkie) [inline src/char/char.c]: Sorry xD going to revert it.

**Comment by dastgir** [comment]
Also please squash the commits

**Comment by MishimaHaruna** [comment]
Thank you, merged as 138a866

**Pull request closed without merging**